src/Swarm/Language/Capability.hs

```diff
@@ -136,27 +59,8 @@ data Capability
     CLambda
   | -- | Enable recursive definitions
     CRecursion
-  | -- | Execute the 'Reprogram' command
-    CReprogram
-  | -- | Execute the `meet` and `meetAll` commands.
-    CMeet
   | -- | Capability to introspect and see its own name
     CWhoami
```

Comment:
Oh, never mind, I see this capability is for both `whoami` and `self` commands.

Reply:
Yes, but perhaps we can have two self explanatory capabilities or have self implied by whoami.

Reply:
Sure, it should always be OK to (1) split out capabilities like this into individual capabilities for each command, then (2) edit any devices which currently provide the old combined capability so that they now provide both.  The only reason not to do that before was to prevent a proliferation of lots of capabilities, but now that we have `CExecute` that's not really an issue.

In fact, it's probably a good idea to just split all such capabilities since it gives more fine-grained control for scenario authors.  For example, the `time` capability used to allow running both `time` and `wait` commands, and we split it into `timeabs` and `timerel` so that we could have scenarios that allow you to run `wait` but not `time`.

Reply:
Done! That took a lot of effort, but our test coverage helped a lot. 😄